src/main/kotlin/nl/avisi/structurizr/site/generatr/site/C4PlantUmlExporterWithElementLinks.kt

```diff
@@ -30,7 +49,8 @@ class C4PlantUmlExporterWithElementLinks(
         workspace.model.includedSoftwareSystems.contains(this) && this != view?.softwareSystem
```

Comment:
This can be changed to:

```suggestion
        this != view?.softwareSystem && this is SoftwareSystem && this.location != Location.External
```

With this change, this class doesn't need `Workspace` anymore.

Reply:
yeah, nice suggestion. Done!